fibo/src/components/VisNetwork.vue

```diff
@@ -43,17 +43,67 @@
         />
         <label class="form-check-label" for="non_optional">required</label>
       </div>
+      <div class="form-check form-check-inline">
+        <button
+        type="button"
+        class="btn btn-link"
+        v-on:click="swapGraphContent()"
+      >
+        expand...
+      </button>
+      </div>
+    </div>
+
+    <div id="smallGraph">
+      <div id="ontograph"></div>
     </div>
 
-    <div id="ontograph"></div>
+    <div
+      class="modal down"
+      id="graphModal"
+      tabindex="-1"
+      role="dialog"
+      aria-labelledby="graphModalLabel"
+      aria-hidden="true"
+      v-bind:class="{ fade: !modalVisible }"
+    >
+      <div
+        class="modal-dialog-full-width modal-dialog momodel modal-fluid"
+        role="document"
+      >
+        <div class="modal-content-full-width modal-content">
+          <div class="modal-header-full-width modal-header">
+            <h5 class="modal-title w-100" id="graphModalLabel">
+              <b class="mr-1 ml-4">Data model for "<i>${clazz.label}</i>"</b>
```

Comment:
On page [BE/Corporations/Corporations/BoardAgreement](http://pr-90.spec.edmcouncil.org/fibo/ontology/BE/Corporations/Corporations/BoardAgreement?version=master%2F2021Q2) I clicked "expand...." and a window appeared with _Data model for "**${clazz.label}**"_
![image](https://user-images.githubusercontent.com/21159504/124000622-0d0a3900-d9d4-11eb-9b62-6fbbf6871351.png)

Reply:
Also @Karpas could you make 'expand' button more prominent/visible and more self-explanatory, e.g. "Expand diagram" or "Diagram in a new window" or something like that?